src/unit-tests/loginAndLogout.test.js

```diff
@@ -0,0 +1,34 @@
+import { login } from "../js/api/auth/login.js";
+import { logout } from "../js/api/auth/logout.js";
+const mockLocalStorage = (() => {
+  let localStorage = {};
+  let localStorageMethods = {
+    getItem: (key) => localStorage[key],
+    setItem: (key, value) => (localStorage[key] = value),
+    removeItem: (key) => delete localStorage[key],
+    clear: () => (localStorage = {}),
+  };
+  return localStorageMethods;
+})();
+const email = "test12345@noroff.no";
+const password = "abc12345";
+global.localStorage = mockLocalStorage;
```

Comment:
This test is failing on my local. You need to have a  look and make it pass. It is required to pass the CA

Reply:
I have now mock fetch so the test will not fail because of random connection errors, the test should work now